client/src/__tests__/unit-tests/driver/bitbake-driver.test.ts

```diff
@@ -76,4 +77,24 @@ describe('BitbakeDriver Tests', () => {
     const script = driver.composeBitbakeScript('bitbake busybox')
     expect(script).toEqual(expect.stringContaining("docker run --rm -it -v /home/user/yocto:/workdir crops/poky --workdir=/workdir /bin/bash -c '. /home/user/yocto/poky/oe-init-build-env && bitbake busybox'"))
   })
+
+  describe('composeBitbakeCommand', () => {
+    let bitbakeDriver: BitbakeDriver
+    beforeEach(() => {
+      bitbakeDriver = new BitbakeDriver()
+    })
+
+    it('should compose bitbake command for scanning recipe environment', () => {
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+      const bitbakeTaskDefinition = {
```

Comment:
client/package.json should reflect the new option. It should also be handled in bitbakeTaskProvider.
Also it's not called by a bitbake command yet?

Reply:
I only test if the command is properly generated.